### ISSUE TITLE
Simple sortable example error while random text dropped

### ISSUE THREAD

**Issue by harrychou** [akserg/ng2-dnd#13]
First of all, excellent work!!

On the Drag and Drop example page (http://akserg.github.io/ng2-webpack-demo/#/dnd), the 'Simple sortable' example will error out if I randomly select some text on the page and drop into the sortable container.

This is for ng2-dnd version 1.4.1. 

Error message: 
==============================================
TypeError: Cannot read property 'next' of undefined(…)
!!! Suggestion: Seems you forget add <ng2-slim-loading-bar></ng2-slim-loading-bar> into your html?
Angular 2 is running in the development mode. Call enableProdMode() to enable the production mode.
EXCEPTION: TypeError: Cannot read property 'undefined' of undefined
EXCEPTION: TypeError: Cannot read property 'undefined' of undefined
STACKTRACE:
TypeError: Cannot read property 'undefined' of undefined
    at SortableComponent._onDragEnterCallback (dnd.sortable.ts:135)
    at SortableComponent.AbstractComponent._onDragEnter (dnd.component.ts:104)
    at HTMLLIElement.AbstractComponent._elem.ondragenter (dnd.component.ts:46)
    at HTMLLIElement.wrapFn [as _ondragenter] (zone.js:748)
    at ZoneDelegate.invokeTask (zone.js:341)
    at Object.NgZoneImpl.inner.inner.fork.onInvokeTask (ng_zone_impl.ts:54)
    at ZoneDelegate.invokeTask (zone.js:340)
    at Zone.runTask (zone.js:238)
    at HTMLLIElement.ZoneTask.invoke (zone.js:408)
Uncaught TypeError: Cannot read property 'undefined' of undefined

**Comment by akserg** [comment]
Hi,
I will have a look at fix it at the rest of the week.

**Comment by jimitndiaye** [comment]
I'm having this same error in version 1.4.1.
I've got a component with the following template:
```html
  <ion-list class="component-list"
    *ngIf="isDesignTime"
    dnd-droppable 
    [dropZones]="['page', 'components', 'component-list']" 
    (onDropSuccess)="dropComponent($event)"
    dnd-sortable-container
    [sortableData]="components">
    <ion-item 
      *ngFor="#component of components; #i = index" 
      dnd-sortable 
      [sortableIndex]="i"
      >
      <dialog-component [config]="component" ></dialog-component>
    </ion-item>
  </ion-list>
  <ion-list class="component-list"
    *ngIf="!isDesignTime">
    <ion-item *ngFor="#component of components" >
      <dialog-component [config]="component" ></dialog-component>
    </ion-item>
  </ion-list>
```
When I drag an item onto the container I get the error:
```
browser_adapter.ts?d90d:73 TypeError: Cannot read property 'undefined' of undefined
    at SortableContainer._onDragEnterCallback (eval at <anonymous> (http://localhost:3000/js/main.bundle.js:1:0), <anonymous>:60:58)
    at SortableContainer.AbstractComponent._onDragEnter (eval at <anonymous> (http://localhost:3000/js/main.bundle.js:1:0), <anonymous>:116:18)
    at HTMLElement.AbstractComponent._elem.ondragenter (eval at <anonymous> (http://localhost:3000/js/main.bundle.js:1:0), <anonymous>:39:19)
    at HTMLElement.wrapFn [as _ondragenter] (eval at <anonymous> (http://localhost:3000/js/polyfills.bundle.js:1:0), <anonymous>:769:30)
    at ZoneDelegate.invokeTask (eval at <anonymous> (http://localhost:3000/js/polyfills.bundle.js:1:0), <anonymous>:356:38)
    at Object.NgZoneImpl.inner.inner.fork.onInvokeTask (eval at <anonymous> (http://localhost:3000/js/vendor.bundle.js:1:0), <anonymous>:36:41)
    at ZoneDelegate.invokeTask (eval at <anonymous> (http://localhost:3000/js/polyfills.bundle.js:1:0), <anonymous>:355:43)
    at Zone.runTask (eval at <anonymous> (http://localhost:3000/js/polyfills.bundle.js:1:0), <anonymous>:256:48)
    at HTMLElement.ZoneTask.invoke (eval at <anonymous> (http://localhost:3000/js/polyfills.bundle.js:1:0), <anonymous>:423:34)
```

**Comment by akserg** [comment]
Fixed in v 1.6.0
Please check and confirm.

**Comment by akserg** [comment]
Sorry, my bad - problem with migration to RC1. I will inform you when I will finish migration.

**Comment by akserg** [comment]
Fixed in v 1.6.4
